nestkernel/dynamicloader.cpp

```diff
@@ -153,11 +153,6 @@ DynamicLoaderModule::LoadModuleFunction::execute( SLIInterpreter* i ) const
 {
   i->assert_stack_load( 1 );
 
-  if ( kernel().model_manager.has_user_models() or kernel().model_manager.has_user_prototypes() )
```

Comment:
Why is this check no longer needed?

Reply:
I really dug deep to understand why this check was there in the first place. Other than a comment by @mschmidt87 in our old trac that "problems might occur when these are not checked" I could not find a good reason. So I removed them together with the pertaining tests.

Reply:
OK, no risk of model IDs getting mixed up when we have arbitrary mixes of CopyModel and ModuleInstall?

Reply:
I don't see such a risk.